src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hover/VSHover.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using HoverModel = OmniSharp.Extensions.LanguageServer.Protocol.Models.Hover;
```

Comment:
I'd remove only because it adds a little bit of overhead

Reply:
Gah whoops, I was wrong - I thought I remembered it being unambiguous when testing before, but it looks like it is (since the namespace is also named Hover). So looks like we may have to do it this way. :(